src/model/device/network.rs

```diff
@@ -202,6 +202,7 @@ pub struct NetworkInterfaceStatus {
     pub proxy_config: ProxyConfig,
     pub domain: Option<String>,
     pub cost: u8,
+    pub errors: Option<Vec<String>>,
```

Comment:
Why option? If empty vec could be no errors

Reply:
@uncleDecart easier to use things like and_then or or_else